### ISSUE TITLE
Error when importing payment profile

### ISSUE THREAD

**Issue by andrewspinks** [maxio-com/ab-python-sdk#45]
I am receiving an error when attempting to import a payment profile with stripe.
```
File "xxxx\Lib\site-packages\apimatic_core\types\union_types\one_of.py", line 32, in validate
    self.error_messages = UnionTypeHelper.process_errors(value, self._union_types, self.error_messages,
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "xxxx\Lib\site-packages\apimatic_core\utilities\union_type_helper.py", line 228, in process_errors
    UnionTypeHelper.raise_validation_exception(value, union_types, ', '.join(error_messages), is_for_one_of)
  File "xxxx\Lib\site-packages\apimatic_core\utilities\union_type_helper.py", line 249, in raise_validation_exception
    raise OneOfValidationException('{} \nActual Value: {}\nExpected Type: One Of {}.'.format(
apimatic_core.exceptions.oneof_validation_exception.OneOfValidationException: We could not match any acceptable types against the given JSON.
Actual Value: {'id': 60550113, 'first_name': 'xxx', 'last_name': 'xxx', 'masked_card_number': None, 'card_type': None, 'expiration_month': None, 'expiration_year': None, 'customer_id': 81902223, 'current_vault': 'stripe_connect', 'vault_token': 'cus_xxx', 'billing_address': None, 'billing_city': None, 'billing_state': None, 'billing_zip': None, 'billing_country': None, 'customer_vault_token': None, 'billing_address_2': None, 'site_gateway_setting_id': 27084, 'payment_type': 'credit_card', 'disabled': False, 'gateway_handle': 'stripe-handle'}
```
I am creating the payment profile as follows:
```
body = CreatePaymentProfileRequest(
    payment_profile=CreatePaymentProfile(
        payment_type=PaymentType.CREDIT_CARD,
        current_vault=CurrentVault.STRIPE_CONNECT,
        vault_token='cus_xxxx',
        customer_id=chargify_customer.customer.id,
    )
  )

  result = client.payment_profiles.create_payment_profile(
      body=body
  )
```
The Payment Method is created in Chargify successfully, but seems to be a problem when parsing the result. I have tried this with both 4.0.0 and 3.0.1.

**Comment by andrewspinks** [comment]
It does work successfully if I also supply 
```
last_four=
expiration_month=
expiration_year=
```
are these required fields?

**Comment by maciej-nedza** [comment]
Thanks for reporting the issue @andrewspinks. Seems like it's a bug on our end. It was caused by missing  `masked_card_number`, which usually is generated at the backend and here it wasn't which we didn't anticipate. We will tackle it in the upcoming SDK release.
And no, these fields are not required when you're importing card with the token.

**Comment by michalpierog** [comment]
@andrewspinks We have released the 5.0.0 version. This issue should be fixed. There are some breaking changes regarding payment profiles and vaults, see: https://github.com/maxio-com/ab-python-sdk/releases/tag/5.0.0
